ts_scripts/sanity_utils.py

```diff
@@ -19,7 +20,7 @@ def run_markdown_link_checker():
     for mdfile in glob.glob("**/*.md", recursive=True):
         cmd = f"markdown-link-check {mdfile} --config link_check_config.json"
         print(f"## In directory: {os.getcwd()} | Executing command: {cmd}")
-        status = os.system(cmd)
+        status = subprocess.run(cmd)
```

Comment:
The return types of `subprocess.run` and `os.system` are different. Please change the status code check logic. https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess